modules/jaggery-apps/appmgt/src/site/pages/createapplication.jag

```diff
@@ -22,6 +22,10 @@ var site = require("/site/conf/site.json");
 
 (function () {
     var type = request.getParameter('type');
+    var apptype = request.getParameter('apptype');
+    var apptypeIconColor = request.getParameter('apptypeIconColor');
+    var apptypeIconImageClass = request.getParameter('apptypeIconImageClass');
+    var appTypeDisplayName = request.getParameter('appTypeDisplayName');

```

Comment:
This is wrong. We should not get apptypeIconColor ,apptypeIconImageClass ,appTypeDisplayName from the request. Those things should be taken from the apptype bean. 

https://wso2.org/jira/browse/APPFAC-3233